### PR TITLE
Added workload for tilesets and objects

### PR DESCRIPTION
**Pull request by kerrielake**
NPC Update:
-Added Joe Portrait
-Added Joe spritesheet
-Added Joe Icon
-Created joe.json (copied from Corey's json file)

Pythonic Temple Update:
-Added Tileset_PythonicTemple.png

API Update:
-Added Tileset_API-Exterior.png
-Added Tileset_API-Interior.png
-Added object "fountain"
-Added object "candle-desk"
-Added object "candle-wall"
-Added object "door-dungeon-left"
-Added object "door-dungeon-right"

### CODE REVIEW

File: levels/fog_owl/maps/default.json [hunk 1]
```diff
@@ -1006,6 +1006,10 @@
         {
          "firstgid":10795,
          "source":"..\/..\/..\/..\/.bundled\/tilesets\/TwilioQuestObjects.json"
+        }, 
+        {
+         "firstgid":11811,
+         "source":"..\/..\/..\/tilesets\/TwilioQuestObjects.json"
```
Comment (rmkubik) [inline levels/fog_owl/maps/default.json:1012]: The Fog Owl should not be referencing the direct TwilioQuestObjects tileset. It should only reference the bundled version above.

File: objects/door-dungeon-left/config.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      if (self !== event.target) {
+        // Do nothing if this interaction event isn't our object
+        return;
+      }
+    },
+  },
+  state: {
+    // closed, closing, open, opening
+    current: "closed",
+  },
```
Comment (rmkubik) [inline objects/door-dungeon-left/config.js:39]: We can delete the state and events keys above from this file. Lines 28 to 39.

This code is all un-used and not doing anything right now. We can implement these objects as needed later on.

File: objects/door-dungeon-right/config.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      if (self !== event.target) {
+        // Do nothing if this interaction event isn't our object
+        return;
+      }
+    },
+  },
+  state: {
+    // closed, closing, open, opening
+    current: "closed",
+  },
```
Comment (rmkubik) [inline objects/door-dungeon-right/config.js:39]: We can delete state and events here as well.

**Review by rmkubik — CHANGES_REQUESTED** [review]

File: levels/fog_owl/maps/default.json (hunk 1 above)
Comment (rmkubik) [reply levels/fog_owl/maps/default.json:1012]: @kerrielake This tileset still seems to be referencing the non-bundled TwilioQuestObjects tileset. The tileset above is correctly referencing the .bundled version.

Comment (kerrielake) [reply levels/fog_owl/maps/default.json]: > @kerrielake This tileset still seems to be referencing the non-bundled TwilioQuestObjects tileset. The tileset above is correctly referencing the .bundled version.

Ok I think I discovered what happened. I was testing out the new object additions on the map, and it must have continued to reference the non-bundled version even after I removed them. I've updated the directory to .bundled in the json file, let me know if that has fixed the issue!

**Review by rmkubik — APPROVED** [review]

**Pull request merged**